File: apps/s-2-admin/src/api/attendee/AttendeeUpdateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type AttendeeUpdateInput = {
  email?: string;
  name?: string;
  timeZone?: string;
  locale?: string | null;
  booking?: BookingWhereUniqueInput | null;
};

File: apps/cactus-demo-dotnet/src/APIs/Attendee/Dtos/AttendeeUpdateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class AttendeeUpdateInput
{
    public int? Id { get; set; }

    public string? Email { get; set; }

    public string? Name { get; set; }

    public string? TimeZone { get; set; }

    public string? Locale { get; set; }

    public int? Booking { get; set; }
}
